### PR TITLE
updated relu function docstring

### PR DESCRIPTION
**Pull request by GrtSid**
This addresses issue #26211 . Added description, example and updated parameter description.

### CODE REVIEW

**Comment by kyscg** [comment]
Duplicate of #26257

**Comment by GrtSid** [comment]
Hi, @kyscg the issue is yet opened and have a look at the code, I have not copied it. The changes made by me are my original and as per the guidelines.

File: tensorflow/python/keras/activations.py [hunk 1]
```diff
@@ -143,22 +143,38 @@ def softsign(x):
...
   `f(x) = x` for `threshold <= x < max_value`,
   `f(x) = alpha * (x - threshold)` otherwise.
 
   Arguments:
-      x: A tensor or variable.
-      alpha: A scalar, slope of negative section (default=`0.`).
-      max_value: float. Saturation threshold.
-      threshold: float. Threshold value for thresholded activation.
+      `x`: A tensor or variable.
+      `alpha`: A float, governs as the slope of function when the input value is less than threshold value, small positive value corresponds to `Leaky RelU` (default=`0.`).
```
Comment (alextp) [inline tensorflow/python/keras/activations.py]: the linter will complain about this line being too long

File: tensorflow/python/keras/activations.py [hunk 2]
```diff
@@ -143,22 +143,38 @@ def softsign(x):
...
+
+  Example:
+      `python
+      Code:
+      a = tf.constant([1.0, -0.5, 3.4, 2.1, 0.0, -6.5], dtype = tf.float32)
+      b=tf.keras.activations.relu(a,alpha=1.0,max_value=3.0,threshold=1.2)
+      print ("Output: ", (b))
+
+      Output:
+      tf.Tensor([-0.20000005 -1.7         3.          2.1        -1.2
```
Comment (alextp) [inline tensorflow/python/keras/activations.py]: We can pretty-print this and reduce the number of digits for -0.2.

We also probably want a more realistic example with alpha=0 and threshold=0 as that's what most people want

**Review by alextp — CHANGES_REQUESTED** [review]

File: tensorflow/python/keras/activations.py (hunk 1 above)
Comment (GrtSid) [reply tensorflow/python/keras/activations.py]:
```suggestion
      `alpha`: A float, governs as the slope of function when the input value is less than threshold value, small positive value corresponds to `Leaky RelU` (default=`0.`).
```

Comment (GrtSid) [reply tensorflow/python/keras/activations.py]: > the linter will complain about this line being too long

Hey, Thank you for your reviews. Please check for the changes as asked by you.

**Review by alextp — APPROVED** [review]

**Comment by mellanox-github** [comment]
Can one of the admins verify this patch?

**Comment by rthadur** [comment]
@GrtSid can you please check build failures ?

**Comment by rthadur** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!

**Pull request closed without merging**